prime-router/src/main/kotlin/fhirengine/utils/FHIRBundleHelpers.kt

```diff
@@ -116,6 +118,40 @@ fun Bundle.addProvenanceReference() {
     }
 }
 
+/**
+ * Return true if Bundle contains an ELR in the MessageHeader.
+ *
+ * @return true if has a MesssageHeader that contains an R01 or ORU_R01, otherwise false.
+ */
+fun Bundle.isElr(): Boolean {
+    var isElr = false
+    val code = FhirPathUtils.evaluate(
+        null,
+        this,
+        this,
+        "Bundle.entry.resource.ofType(MessageHeader).event.code"
+    )
+        .filterIsInstance<CodeType>()
+        .firstOrNull()
+        ?.code
+    if (code != null && ((code == "R01") || (code == "ORU_R01"))) {
```

Comment:
nit: you can just return this expression rather using the mutable `isElr` variable.

Reply:
and you can also skip the null check since `null == "RO1` will be false as well

Reply:
Noice!  Made the change.